extensions/amp-anim/amp-anim.md

```diff
@@ -27,7 +27,7 @@ limitations under the License.
   </tr>
   <tr>
     <td class="col-fourty"><strong><a href="https://www.ampproject.org/docs/guides/responsive/control_layout.html">Supported Layouts</a></strong></td>
-    <td>fill, fixed, fixed-height, flex-item, nodisplay, responsive</td>
+    <td>fill, fixed, fixed-height, flex-item, intrinsic, nodisplay, responsive</td>
```

Comment:
There are technically a lot more components that can support this given `isFixedLayout()` returns true but I do like limiting documentation for the ones that this makes sense for. I'd add `amp-video` to the current list you have as well.

Reply:
I'm going to add amp-ad to fix #12261